connectors/src/connectors/github/temporal/workflows.ts

```diff
@@ -240,16 +240,15 @@ export async function githubRepoSyncWorkflow(
     await Promise.all(promises);
   }
 
-  // Start child workflow to sync code.
-  const childWorkflowId = `${workflowInfo().workflowId}-codeSync`;
-  await executeChild(githubCodeSyncWorkflow, {
-    workflowId: childWorkflowId,
-    searchAttributes: {
-      connectorId: [parseInt(connectorId)],
-    },
-    args: [dataSourceConfig, githubInstallationId, repoName, repoId, repoLogin],
-    parentClosePolicy: ParentClosePolicy.PARENT_CLOSE_POLICY_TERMINATE,
-    memo: workflowInfo().memo,
+  // Start code syncing activity.
+  await githubCodeSyncActivity({
```

Comment:
remove `connectorId` from workflow args?

Reply:
Not sure I understand?

Reply:
Ah I presume you mean that I added it and now can be removed?

Reply:
Will do in follow-up PR